### DISCUSSION TITLE
two computers with same Chia account

### DISCUSSION THREAD

**Discussion by chianew** [Chia-Network/chia-blockchain#2905] [Q&A]
Hello, I'm new to chia. Can I use different computer with one wallet account adding plots and farming on a LAN at same time?

Thank you very much!

**Comment by dorkmo** [comment]
if you use the same 24 word key they will farm for the same wallet. if you are on the same network, it is best to run one computer with a node and the other as only a harvester pointed at the node

**Reply by chianew** [reply]
thank you! If I use different IP, can I use the same 24 word key on both computers adding plot and farming ?

**Reply by cccat6** [reply]
You can do that. Just remind don't farm on the same plot. Double farming and copy the plot doesn't work.
